the-collab-lab/tcl-70-smart-shopping-list
# Refactored header, fixed mobile navigation and added background image for main section

## Description
 
In the header, there should be an img as well an h1 instead of just an empty h1 and a background image. The text does not look exactly like the background image that was there before because the font in the background image was not a available. The word `SHOP` is also not curved but will be fixed in a future issue.

On the list page the list items no longer overlap the bottom navigation like it did before.

The main sections of all pages should have a background image.


## Related Issue

closes #50 

## Acceptance Criteria

- [x] The header should no longer be just a background image but the image and title (h1)  should be responsive elements on the page.
- [x] The mobile bottom navigation should be on top of all elements in mobile view
- [x] The mobile bottom navigation background color and text color should match the navbar of larger screens
- [x] The mobile navigation icons should match the name of the page.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancements     |


## Updates

### Before
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/36affb3c-ae63-4914-ae1c-486811fe744d)

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/67e50951-de76-444e-bd4f-fafa17cb366d)


### After

![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/82ca0e6d-f3fc-4dcc-98fc-e2cb53d0d379)
![image](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/28488515/5a0a8ae5-fd0a-4d33-97ea-ef993d490571)


## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin km-refactor-header` and check that branch out with `git checkout km-refactor-header` 
* Sign in to view changes
